docs/usage/config-presets.md

```diff
@@ -43,10 +43,11 @@ You can set a Git tag (like a SemVer) to use a specific release of your shared c
 ### GitHub
 
 | name                                        | example use                      | preset    | resolves as                  | filename        | Git tag        |
-| ------------------------------------------- | -------------------------------- | --------- | ---------------------------- | --------------- | -------------- |
+|---------------------------------------------|----------------------------------|-----------|------------------------------|-----------------|----------------|
```

Comment:
i think you need to add the starting and ending spaces again to make prettier happy

Reply:
@HonkingGoose do you have any tips on auto formatting tables after you make updates?

Reply:
prettier should do it automatically

Reply:
I think Husky runs Prettier automatically before committing.

Or they can run `yarn doc-fix` manually.

Reply:
It would do it automatically, if I had a need to have the project setup for development; this is was the default table formatting that emacs did when I updated the guide.  Updated with the yarn target provided.